parser.go

```diff
@@ -103,6 +104,17 @@ func ParseSSHConfig(path string) ([]*SSHHost, error) {
 	return parse(string(content))
 }
 
+// Parse parses a SSH config given by path contained in fsys.
+func Parse(fsys fs.FS, path string) ([]*SSHHost, error) {
```

Comment:
Maybe call it `ParseSSHConfigFS` or so, so it's similarly named as the existing functions?

Reply:
I think `Parse` is better, since otherwise it will be used as sshconfig.ParseSSHConfigFS, and the repeated sshconfig part seems useless.

But, your repo, youe choice! If you prefer, I'll rename it, no problem.

Reply:
Good point. Then I would call it `ParseFS` or so, I don't know if there are any conventional naming for versions that take `fs.FS` yet? I would expect `Parse()` to be the version only taking a `path string` as argument.

Reply:
`ParseFS` seems fine, I don't think there's an accepted convention yet... maybe we establish one 😀

Would you want me to also rename `ParseSSHConfigFS` to `Parse`, and add a deprecated
`ParseSSHConfigFS` stub that just call `Parse`? 

Since I have to add a mod file because go 1.16 requires it, it would be nice if you push a 1.0.0 version afterwards, so that we could depend on nice tagged versions...

Reply:
Yes, please do. I can tag a version after, but would probably be v0.x.x as I wouldn't want v1.0.0 with straight away deprecated public functions :)

Appreciate the effort from your side!

Reply:
Ok, I'll do...

> Appreciate the effort from your side!

You're welcome, I'll use `sshconfig` in production working, 
I'm just giving back... 

Your code will hopefully run on https://lexis-project.eu/web/ within some months, you spared me a fair amount of work. Thank you...